zchronod/network/src/rpc.rs

```diff
@@ -0,0 +1,44 @@
+use tonic::{transport::Server, Request, Response, Status, IntoRequest};
+use log::info;
+use log::kv::ToKey;
+use api::CONTEXT;
+use proto::zchronod::zchronod_server::{Zchronod, ZchronodServer};
+use proto::zchronod::{ZchronodRequest, ZchronodResp};
+
+pub(crate) struct RpcServer {
+    port: String,
+}
+
+impl RpcServer {
+    pub(crate) fn new(listen: &str) -> Self {
+        RpcServer {
+            port: listen.to_string(),
```

Comment:
confused arg "listen",what it means? ipaddr or Port? need more clear defination

Reply:
change as self_address